### ISSUE TITLE
Importing relative paths broken in 1.1.1

### ISSUE THREAD

**Issue by deisner** [chriseppstein/sass-globbing#19]
A statement like `@import "../variables/**/*";` works fine with sass-globbing v1.1.0, but it appears to be broken in v1.1.1.

Here's an example that demonstrates the problem. With 1.1.0, everything is fine:
```shell
$ find .
.
./css
./css/layouts
./sass
./sass/variables
./sass/variables/_colors.scss
./sass/layouts
./sass/layouts/test.scss

$ cat sass/variables/_colors.scss
$blue: #22F;

$ cat sass/layouts/test.scss
@import "../variables/**/*";

body {
  background: $blue;
}

$ gem list sass-globbing

*** LOCAL GEMS ***

sass-globbing (1.1.0)

$ sass -r sass-globbing --update sass:css
      write css/layouts/test.css

$ cat css/layouts/test.css
body {
  background: #2222ff; }
```

However, in v1.1.1, I get an error:
```shell
$ gem list sass-globbing

*** LOCAL GEMS ***

sass-globbing (1.1.1, 1.1.0)

$ sass -r sass-globbing --update sass:css
      error sass/variables/**/* (Line 1: File to import not found or unreadable: ../variables/_colors.scss.
Load paths:
  /tmp/bug/sass
  Sass::Globbing::Importer)
$
```

**Comment by renestalder** [comment]
@himynameisdave's solution worked for me too.

**Comment by salacis** [comment]
@himynameisdave @renestalder , this turns out to work for me, too!

**Comment by erikphanson** [comment]
@himynameisdave worked for me as well. Back on the 1.1.0

**Comment by willviles** [comment]
+1 for sass-globbing-pr-20 solving the problem

**Comment by chriseppstein** [comment]
Fixed in `1.1.3`
